pysqldb3/pysqldb3.py

```diff
@@ -2085,25 +2085,26 @@ def feature_class_to_table(self, path, table, schema=None, shp_name=None, gdal_d
             self.__run_table_logging([schema + "." + table], days=days)
 
 
-    def query_to_gpkg(self, query, path=None, gpkg_name=None, cmd=None, gpkg_tbl = None, gdal_data_loc=GDAL_DATA_LOC,
+    def query_to_gpkg(self, query, gpkg_tbl = None, path=None, gpkg_name=None, cmd=None,  gdal_data_loc=GDAL_DATA_LOC,
                      print_cmd=False, srid=2263):
         """
         Exports query results to a geopackage (.gpkg) file.
         :param query: SQL query as string type
-        :param path: folder path for output gpkg
+        :param gpkg_tbl (str): Table name to be written in the Geopackage output
```

Comment:
doesnt gpkg_name need to be non-optional for this?

Reply:
There is a test called `test_query_to_gpkg_basic_pth` based on `test_query_to_shp_basic_pth` that tests for whether the function still works if a full path is given instead of a gpkg name.  So I allowed `gpkg_name` not to be written so that it would pass this test. Maybe we want to remove this functionality?